samples/quote/Makefile

```diff
@@ -0,0 +1,33 @@
+# Copyright © 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+ACONCLI = aconcli
+OPENSSL = openssl
+DOCKER  = docker
+GO      = go
+
+all: server.json client/sampleclient
+
+server.json: server/Dockerfile signer.pem signer.cer | .acon/
+	$(DOCKER) build -t sampleserver -f $< $(PWD)/../..
+	$(ACONCLI) generate -i sampleserver $@
```

Comment:
I remember that "writableFS" needs to be changed to "true".

Reply:
Does quote_server need to write disk files? If no, then we don't need writable FS.

Reply:
No. Seems there is a bug of acond.

Reply:
@binxing, is `/run/user/0` a must directory for readonly file system?

Reply:
`/run` is a separate tmpfs mount that shouldn't be affected by `writableFS`.

Reply:
I mean `/run/user/*`, not `/run`.